### DISCUSSION TITLE
[Feature] day description

### DISCUSSION THREAD

**Discussion by mtettke123** [immich-app/immich#2549] [Feature Request]
### The feature

Managing photos on my PC "the old way", I used a directory structure like "YYYY-MM-DD <description>".

It would be great if immich could offer a functionality to enter a short "day" description (not only to individual images how it is planks now) that could be used in the storage path. Would (nearly) complete the functionality for me...

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Pheggas** [comment]
The best way is to add new album type IMHO. So it end up being an album with all the photos from one entire day and there would be ability to add a describtion to it. But i'm not sure how many users would use it.
